FIPS/src/test/java/net/snowflake/client/TestUtil.java

```diff
@@ -47,7 +47,11 @@ public interface TestRunInterface {
    */
   public static String systemGetEnv(String env) {
     try {
-      return System.getenv(env);
+      String value = System.getenv(env);
```

Comment:
why do we need to check both places now?

Reply:
@sfc-gh-dprzybysz When I was executing the test for Windows, although the environment variable was set when we use System.getenv it return null so If the value is null it will try with the backup mechanism extracting the value from the map returned by System.getenv()

Reply:
were you able to investigate it? I don't think that both methods should be different - especially that it would mean that on Windows the driver code behaves differently even now

Reply:
@sfc-gh-dprzybysz  Can I resolve this comment?